opentech/apply/review/forms.py

```diff
@@ -78,45 +79,29 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def validate_unique(self):
-        super().validate_unique()
-
+        # update the instance data prior to validating uniqueness
         self.instance.submission = self.submission
         self.instance.author = self.request.user
         self.instance.review = self.cleaned_data
 
-        try:
-            self.instance.validate_unique()
-        except ValidationError as e:
-            self._update_errors(e)
+        super().validate_unique()
 
     def save(self, commit=True):
+        self.instance.score = self.calculate_score()
 
-        score_fields = self.get_score_fields()
+        self.instance.recommendation = self.cleaned_data['recommendation']
```

Comment:
I like to be defensive.

Reply:
`recommendation` is required. if it isnt in `cleaned_data` we have bigger problems